Makefile

```diff
@@ -184,6 +184,17 @@ test.integration.dbless:
 		-coverprofile=coverage.dbless.out \
 		./test/integration
 
+.PHONY: test.integration.enterprise.dbless
+test.integration.enterprise.dbless:
+	@./scripts/check-container-environment.sh
+	@TEST_DATABASE_MODE="off" GOFLAGS="-tags=integration_tests" go test -v -race \
+		-timeout 15m \
+		-parallel $(NCPU) \
+		-covermode=atomic \
+		-coverpkg=$(PKG_LIST) \
+		-coverprofile=coverage.enterprisedbless.out \
```

Comment:
We should add the coverage produced here to `test.yaml` so that it can be collected for our coverage step.

Reply:
added https://github.com/Kong/kubernetes-ingress-controller/pull/1758/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692R140